api/mapping/views.py

```diff
@@ -968,6 +973,10 @@ class ScanReportFormView(FormView):
     template_name = "mapping/upload_scan_report.html"
     success_url = reverse_lazy("scan-report-list")
 
+    def form_invalid(self, form):
+        error_dict= {'status':'form-invalid','form-errors':form.errors}
+        return HttpResponse(json.dumps(error_dict),content_type="application/json")
+
```

Comment:
Is this the bit that gives back the errors for your screenshot on the PR?

Reply:
Yes it is